src/components/trade/LastTrades/LastTradesWidget.tsx

```diff
@@ -42,7 +55,7 @@ function toLastTradeItem(trade: any): LastTradesItem {
 }
 
 export const LastTradesWidget: React.FC<Props> = (props) => {
-  const { loading, error, data } = useQuery(GET_TRADES)
+  const { loading, error, data } = useQuery<GET_TRADES_RESULT>(GET_TRADES)
```

Comment:
I love to see this change, but is unrelated to changing typescript to 4.0, right?

Reply:
Related
It went like that `update TS -> update linter plugins for TS -> linter complains even louder about untyped stuff`